### ISSUE TITLE
Minor UX Improvements

### ISSUE THREAD

**Issue by crispeeweevile** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#39]
Currently the scroll speed for zooming in, and also for scrolling in the "load save selection" are both extremely slow, even at max speed. Basically I think the speed should be a bit faster (1.5x-2x) camera move speed seems fine

**Comment by AOx0** [comment]
Thanks. We will look into it.

**Comment by AOx0** [comment]
Question, what are you using to zoom in and to scroll in the Save selection? Speed is decent for me with my laptop's Track-pad 

https://user-images.githubusercontent.com/50227494/205545183-16b1dd73-3259-479f-b1c9-7bde3328d60c.mov

Can you show a video of the behavior you are experiencing?

**Comment by crispeeweevile** [comment]
> Question, what are you using to zoom in and to scroll in the Save selection? Speed is decent for me with my laptop's Track-pad
> 
>  Screen.Recording.2022-12-04.at.9.46.59.p.m.mov 
> Can you show a video of the behavior you are experiencing?

I can send a video, got any suggestions on what software I should be using to take said video? Also, I'm using my "gaming" mouse. Could be a setting, I'll check that too.

**Comment by AOx0** [comment]
If you are on macOS then there's a built-in recorder with Shift+Cmd+5

If you are on Windows I'm not sure there's a built-in one but I know Obs can record your screen, same with Linux

**Comment by crispeeweevile** [comment]
> Question, what are you using to zoom in and to scroll in the Save selection? Speed is decent for me with my laptop's Track-pad



https://user-images.githubusercontent.com/92649061/205546714-e71b2d22-ffa4-4dd7-acda-47eb20aad59e.mp4


> 
> Can you show a video of the behavior you are experiencing?

Alrighty- I just used OBS. The zoom speed certainly doesn't seem as bad as the scrolling in the load menu



https://user-images.githubusercontent.com/92649061/205546904-3eba3ad4-94b9-4585-97be-f8391b505e71.mp4

**Comment by AOx0** [comment]
That's definitely slow. Is it with your mouse?

**Comment by crispeeweevile** [comment]
> That's definitely slow. Is it with your mouse?

Yes I'm using my mouse, it's a Red Dragon gaming mouse, I suppose one of my settings could be "bad"
edit: it's not significantly better using my trackpad

**Comment by AOx0** [comment]
I think this needs further discussion, maybe keybindings. Since it's not a hot-fix thing, I'm postponing this to v0.40.0 so I can deploy the fix for #41. 
Thank you very much for the report, and I’m open to more ideas that can potentially solve the issue.

**Comment by HannesGitH** [comment]
my menu was also terribly slow but vsync was set to zero, setting it to 1/1 helped a lot

**Comment by pudy248** [comment]
I could replicate this on Windows and Linux, doesn't appear platform-specific in any case. A 2-3x multiplier to the scroll amount would probably be sufficient for a 1 line fix.
